### PR TITLE
Add selenium tests for headers on edit pages

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

**Comment by Beacze** [comment]
Selenium tests are passing on my local run (I have run them multiply times). It would be nice if someone would also run those tests so we can detect what is wrong.

File: Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProcessesPage.java [hunk 1]
```diff
@@ -47,6 +51,19 @@ public ProcessesPage goTo() throws Exception {
...
+    /**
+     * Go to edit page for creating a new project.
+     *
+     * @return project edit page
+     */
+    public ProcessEditPage editProcess() throws Exception {
+        if (isNotAt()) {
+            goTo();
+        }
+        clickButtonAndWaitForRedirect(editProcessLink, Pages.getProjectEditPage().getUrl());
```
Comment (tim-boerner) [inline Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProcessesPage.java]: Edit process or project? ;-)

File: Kitodo/src/test/java/org/kitodo/selenium/EditingST.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    public void logout() throws Exception {
+        Pages.getTopNavigation().logout();
+        if (Browser.isAlertPresent()) {
+            Browser.getDriver().switchTo().alert().accept();
+        }
+    }
+
+    @Test
+    public void editProcessTest() throws Exception {
+        assumeTrue(!SystemUtils.IS_OS_WINDOWS && !SystemUtils.IS_OS_MAC);
```
Comment (tim-boerner) [inline Kitodo/src/test/java/org/kitodo/selenium/EditingST.java]: That is why its not failing on windows machines, it just doesnt run ;-)

File: Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProcessesPage.java (hunk 1 above)
Comment (Beacze) [reply Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProcessesPage.java]: Thank you! :D

**Pull request merged**